.github/workflows/tests.yml

```diff
@@ -22,71 +22,139 @@ jobs:
   integration:
     name: Integration
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        database: ['sqlite', 'postgres']
 
     steps:
       - name: Checkout code
         uses: actions/checkout@master
+      - name: Set up PostgreSQL
+        if: ${{ matrix.database == 'postgres' }}
+        run: |
+          docker run -d -p 5432:5432 \
+            -e POSTGRES_PASSWORD=postgres \
+            -e POSTGRES_USER=postgres \
+            -e POSTGRES_DB=postgres \
+            postgres:latest
       - uses: actions/setup-node@v2
         with:
           node-version: '16.13.2'
           cache: 'yarn'
       - name: Fetch Node Packages
         run: |
           yarn --immutable
+      - name: Await PostgreSQL
+        if: ${{ matrix.database == 'postgres' }}
+        timeout-minutes: 2
+        run: until pg_isready -h localhost; do sleep 1; done
       - name: Run tests
         run: |
           yarn test:int --coverage --verbose
+        env:
+          DATABASE_URL: ${{ matrix.database == 'postgres' && 'postgres://postgres:postgres@localhost:5432/postgres' || '' }}
 
```

Comment:
Why the `|| ''` at the end there?

Reply:
It's the equivalent of a ternary operator meaning:

`condition && true || false` instead of the classic `condition ? true : false `

Reply:
Otherwise, I think it would set the value of `DATABASE_URL` to `false`